src/components/Input/Input.js

```diff
@@ -43,15 +42,19 @@ const Input = props => {
   }
 
   if (error) {
-    containerStyles.push(styles.containerWithErrors);
     labelStyles.push(styles.labelWithErrors);
   }
 
   const { wrapperStyles } = props;
-
+  const getContainerStyles = () => [
```

Comment:
why don't simply declare an array? do u need this function?

Reply:
I have wanted to separate styles with condition. In my prev project we have used lib classnames, but I not sure about react native project.
Fixed